app/scripts/modules/core/src/managed/versionsHistory/VersionHeading.tsx

```diff
@@ -57,8 +61,16 @@ interface IVersionHeadingProps {
 
 export const VersionHeading = ({ group, chevron }: IVersionHeadingProps) => {
   const gitMetadata = group.gitMetadata;
+  const client = useApolloClient();
+  const app = useApplicationContextSafe();
+  const prefetchData = () => {
+    client.query<any, FetchVersionQueryVariables>({
+      query: FetchVersionDocument,
+      variables: { appName: app.name, versions: Array.from(group.versions) },
+    });
+  };
```

Comment:
is it worth commenting where the results of this query is expected to be consumed?